CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ if (BUILD_SDF)
 
   ########################################
   # Find ignition common
-  ign_find_package(ignition-common5 COMPONENTS graphics REQUIRED_BY usd)
+  ign_find_package(ignition-common5 COMPONENTS graphics REQUIRED)
```

Comment:
Is there a way to require only the core of ign-common and not the graphics component? I think the graphics component brings too many dependencies that we wouldn't want to impose on SDFormat users.

Even the core of ign-common brings a dependency on `libdl`, which I think is an unnecessary constraint for SDFormat. As @chapulina suggested in https://github.com/ignitionrobotics/sdformat/pull/911#discussion_r838737967, perhaps we should move some of the functions used in this PR to `ign-utils`.

Reply:
I've added a find package for the just the main part of common. There are a couple other pull requests that also make use of ign-common's Filesystem and Console.